python/paddle/fluid/dygraph/dygraph_to_static/convert_call_func.py

```diff
@@ -103,63 +103,63 @@ def dyfunc(x):
             return func
         try:
             if func in func.__globals__.values():
-                if six.PY3:
-                    source_code = inspect.getsource(func)
-                    if any(decorator in source_code
-                           for decorator in DECORATOR_NAMES):
-                        converted_call = None
-                    else:
-                        converted_call = to_static_func(func)
-                        func_self = getattr(func, '__self__', None)
-                else:
-                    converted_call = to_static_func(func)
-                    func_self = getattr(func, '__self__', None)
+                # if six.PY3:
+                #     source_code = inspect.getsource(func)
+                #     if any(decorator in source_code
+                #            for decorator in DECORATOR_NAMES):
+                #         converted_call = None
+                #     else:
+                #         converted_call = to_static_func(func)
+                #         func_self = getattr(func, '__self__', None)
+                # else:
+                converted_call = to_static_func(func)
+                func_self = getattr(func, '__self__', None)
```

Comment:
I will remove these code will be removed in next commit.

Reply:
We discussed offline. She split into two case because `inpect.getsource(func)` will raise a `OSError` when `func` is decorated in PY2. In this case, we should use `inspect.getsource(getattr(dyfunc, '__wrapped__'))` instead of `dyfunc` directly.

Reply:
Great